Compilation error with new lv2 release (1.18.0)

```
[ 94%] Building CXX object CMakeFiles/artyfx_ui.dir/src/ui/roomy.cxx.o
/gnu/store/x3jx25cd3q363mr7nbgzrhrv1vza6cf7-gcc-7.4.0/bin/c++  -Dartyfx_ui_EXPORTS -I/tmp/guix-build-artyfx-1.3.drv-0/source/src/avtk/avtk  -O2 -g -DNDEBUG -fPIC   -o CMakeFiles/artyfx_ui.dir/src/ui/roomy.cxx.o -c /tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/roomy.cxx
[ 96%] Building CXX object CMakeFiles/artyfx_ui.dir/src/ui/satma.cxx.o
/gnu/store/x3jx25cd3q363mr7nbgzrhrv1vza6cf7-gcc-7.4.0/bin/c++  -Dartyfx_ui_EXPORTS -I/tmp/guix-build-artyfx-1.3.drv-0/source/src/avtk/avtk  -O2 -g -DNDEBUG -fPIC   -o CMakeFiles/artyfx_ui.dir/src/ui/satma.cxx.o -c /tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/satma.cxx
[ 97%] Building CXX object CMakeFiles/artyfx_ui.dir/src/ui/vihda.cxx.o
/gnu/store/x3jx25cd3q363mr7nbgzrhrv1vza6cf7-gcc-7.4.0/bin/c++  -Dartyfx_ui_EXPORTS -I/tmp/guix-build-artyfx-1.3.drv-0/source/src/avtk/avtk  -O2 -g -DNDEBUG -fPIC   -o CMakeFiles/artyfx_ui.dir/src/ui/vihda.cxx.o -c /tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/vihda.cxx
[ 98%] Building CXX object CMakeFiles/artyfx_ui.dir/src/ui/whaaa.cxx.o
/gnu/store/x3jx25cd3q363mr7nbgzrhrv1vza6cf7-gcc-7.4.0/bin/c++  -Dartyfx_ui_EXPORTS -I/tmp/guix-build-artyfx-1.3.drv-0/source/src/avtk/avtk  -O2 -g -DNDEBUG -fPIC   -o CMakeFiles/artyfx_ui.dir/src/ui/whaaa.cxx.o -c /tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/whaaa.cxx
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
 };
 ^
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
/tmp/guix-build-artyfx-1.3.drv-0/source/src/ui/lv2_ui.cxx:222:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
make[2]: *** [CMakeFiles/artyfx_ui.dir/build.make:300: CMakeFiles/artyfx_ui.dir/src/ui/lv2_ui.cxx.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/tmp/guix-build-artyfx-1.3.drv-0/build'
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/artyfx_ui.dir/all] Error 2
make[1]: Leaving directory '/tmp/guix-build-artyfx-1.3.drv-0/build'
make: *** [Makefile:133: all] Error 2
command "make" "-j" "8" failed with status 2
```

--- a/friza/gui/ui.cxx
+++ b/friza/gui/ui.cxx
@@ -46,7 +46,7 @@ typedef struct {
 	LV2UI_Controller controller;
 } GUI;
 
-LV2UI_Handle friza_instantiate(const struct _LV2UI_Descriptor * descriptor,
+LV2UI_Handle friza_instantiate(const struct LV2UI_Descriptor * descriptor,
                                const char * plugin_uri,
                                const char * bundle_path,
                                LV2UI_Write_Function write_function,

--- a/friza/gui/ui.hxx
+++ b/friza/gui/ui.hxx
@@ -18,7 +18,7 @@
  * MA 02110-1301, USA.
  */
 
-extern LV2UI_Handle friza_instantiate(const struct _LV2UI_Descriptor * descriptor,
+extern LV2UI_Handle friza_instantiate(const struct LV2UI_Descriptor * descriptor,
                                       const char * plugin_uri,
                                       const char * bundle_path,
                                       LV2UI_Write_Function write_function,

--- a/src/avtk/lv2/testUi.cxx
+++ b/src/avtk/lv2/testUi.cxx
@@ -10,7 +10,7 @@
 #include "../test_ui.hxx"
 
 
-static LV2UI_Handle avtk_instantiate(const struct _LV2UI_Descriptor * descriptor,
+static LV2UI_Handle avtk_instantiate(const struct LV2UI_Descriptor * descriptor,
                                      const char * plugin_uri,
                                      const char * bundle_path,
                                      LV2UI_Write_Function write_function,

--- a/src/ui/lv2_ui.cxx
+++ b/src/ui/lv2_ui.cxx
@@ -33,7 +33,7 @@
 #include "whaaa.hxx"
 
 
-static LV2UI_Handle artyfx_instantiate(const struct _LV2UI_Descriptor * descriptor,
+static LV2UI_Handle artyfx_instantiate(const struct LV2UI_Descriptor * descriptor,
                                        const char * plugin_uri,
                                        const char * bundle_path,
                                        LV2UI_Write_Function write_function,

